dissect/target/loaders/mqtt.py

```diff
@@ -411,6 +418,8 @@ def connect(self) -> None:
 @arg("--mqtt-ca", dest="ca", help="certificate authority file")
 @arg("--mqtt-command", dest="command", help="direct command to client(s)")
 @arg("--mqtt-diag", action="store_true", dest="diag", help="show MQTT diagnostic information")
+@arg("--mqtt-username", dest="username", help="Username for connection")
+@arg("--mqtt-password", dest="password", help="Password for connection")
```

Comment:
Maybe an idea to prompt for the password, so it does not end up on the commandline?